src/lib.rs

```diff
@@ -1,14 +1,10 @@
+//! SSD1306 OLED display driver
+
 #![no_std]
 // TODO: Docs
-// #![deny(missing_docs)]
 // #![deny(missing_debug_implementations)]
-#![deny(missing_copy_implementations)]
-#![deny(trivial_casts)]
-#![deny(trivial_numeric_casts)]
-#![deny(unsafe_code)]
-#![deny(unstable_features)]
-#![deny(unused_import_braces)]
-#![deny(unused_qualifications)]
```

Comment:
Hmm. Can we keep these uncommented guards in? I don't think it's hurting anything at the moment. Rustfmt fixes the issues they bring up which is easy enough to run. Is there a reason you removed them?

Reply:
Yeah my bad. Was using a butcher's knife where I should have been using a scalpel.